### PR TITLE
Update changelog.txt for 4.11 release

### PR DESCRIPTION
**Pull request by trwalke**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+4.11.0
+============
+Feature Requests: **MSAL.NET will now remove accounts from the cache that have expired refresh tokens** MSAL.NET will remove the refresh token and the associated account if the if suberror is "bad_token" to avoid overloading ESTS. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)
+
+Bug Fixes:
+**CryptographicException: Bad PKCS7 padding in iOS brokered authentication** ? [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1628)
```
Comment (trwalke) [inline changelog.txt:6]: @jennyf19 not sure how to explain what is happening here based on whats in the issue, can you elaborate? also, is this the right issue for th iOS fix?

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,11 @@
+4.11.0
+============
+Feature Requests: **MSAL.NET will now remove accounts from the cache that have expired refresh tokens** MSAL.NET will remove the refresh token and the associated account if the if suberror is "bad_token" to avoid overloading ESTS. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)
```
Comment (jmprieur) [inline changelog.txt:3]: **MSAL.NET will now remove accounts from the cache that have expired refresh tokens**. MSAL.NET will remove both the refresh token and the associated account if the if `suberror` is "bad_token" to unnecessary calls to AzureAD. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)

File: changelog.txt (hunk 1 above)
Comment (jmprieur) [reply changelog.txt:6]: Xamarin.iOS Applications using broker, migrated from ADAL.NET to MSAL.NET used to crash. This is now fixed.

Comment (jennyf19) [reply changelog.txt:6]: I would say this:
**When migrating a Xamarin application from ADAL.NET to MSAL.NET and preserving the keychain, a CryptographicException can be thrown from the BrokerKeyHelper.** MSAL.NET now does the broker key keychain look up by Service and Account only. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1628)

---
In reply to: [403313292](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1741#discussion_r403313292) [](ancestors = 403313292)

Comment (jennyf19) [reply changelog.txt:6]: @jmprieur - is this ok?

---
In reply to: [403326603](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1741#discussion_r403326603) [](ancestors = 403326603,403313292)

**Comment by jennyf19** [comment]
thank you travis.

---
In reply to: [387561785](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1741#issuecomment-387561785) [](ancestors = 387561785)

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,18 @@
+4.11.0
+============
+New Features: 
+
+**MSAL.NET will now remove accounts from the cache that have expired refresh tokens**. MSAL.NET will remove both the refresh token and the associated account if the `suberror` is "bad_token" to avaoid unnecessary calls to AzureAD. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)
```
Comment (pmaytak) [inline changelog.txt:5]: typo: 'avaoid' > 'avoid'

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+
+**MSAL.NET uses telemetry schema V2** MSAL.NET has been updated to use Http telemetry schema V2. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1681)
+
+Bug Fixes:
+
+**When migrating a Xamarin application from ADAL.NET to MSAL.NET and preserving the keychain, a CryptographicException can be thrown from the BrokerKeyHelper.** MSAL.NET now does the broker key keychain look up by Service and Account only. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1628)
+
+**WithProofOfPosession produces a token of type POP when it is expected to be PoP** MSAl.NET will now produce a token of type PoP when WithProofOfPosession() is used. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1711)
```
Comment (pmaytak) [inline changelog.txt:13]: typo: 'MSAl.NET' > 'MSAL.NET'

**Comment by jennyf19** [comment]
@trwalke did you see these suggestions from @pmaytak ?

**Pull request merged**